content/blog/end-to-end-easy-to-use-pipeline-for-training-a-model-on-medmnist-v2-using-hpe-machine-learning-development-environment-flask.md

```diff
@@ -0,0 +1,514 @@
+
+![](/img/adipose.jpeg)
+
+`"prediction": "adipose"`
+
+### HPE Machine Learning Development Environment 
+
+On its own, [HPE Machine Learning Development Environment](https://www.hpe.com/us/en/solutions/artificial-intelligence/machine-learning-development-environment.html) helps developers and scientists focus on innovation by removing the complexity and cost associated with at-scale machine learning model training. Today we’re going to explore the advantages of building an end-to-end ML workflow using HPE Machine Learning Development Environment – occasionally referring to HPE’s open source platform, [Determined](http://determined.ai/) - for model training. We also provide an example end-to-end solution using Flask for model deployment, as well as the steps taken to develop this example in the Appendix. 
+
+#### Why use HPE Machine Learning Development Environment? 
+
+At its core, HPE Machine Learning Development Environment is a training platform that reduces complexity for ML researchers and helps research teams collaborate.  
+
+Researchers currently write training scripts that include not only the core ML functionalities to train a model on a dataset, but also code to manage the underlying infrastructure such as training on multiple GPUs, running a hyperparameter search, visualizing the training progress, and saving model checkpoints. Researchers should not have to focus on infrastructure problems – taking away these software engineering and systems administration-related tasks can allow researchers to focus on what’s important: building great models.  
+
+Additionally, collaboration is an important part of ML development. Many research teams don’t have the appropriate resources to share experiment results or share GPU infrastructure, resulting in a lack of reproducibility and ad-hoc resource management. This frustration due to a lack of high-quality resources causes slow progress and is a common reason why ML projects fail.  
+
+In this blog post, we’ll see firsthand how HPE Machine Learning Development Environment can remove infrastructure code in a real-world research script and, at the same time, provide out-of-the-box distributed training, checkpointing, hyperparameter search, and visualization functionality, drastically accelerating research teams’ capabilities. We’ll also touch on the features that allow teams to collaborate effectively. 
+
+If you are interested in more details about how this example was developed, take a look at the Appendix. For a full, in-depth, model porting guide, check out this [model porting guide.](https://docs.determined.ai/latest/tutorials/pytorch-porting-tutorial.html) The code for this example and instructions to run can be found in the [repository](https://github.com/ighodgao/determined_medmnist_e2e). 
+
+#### Experiment Visualization and Metric Logging 
```

Comment:
To be sentence case, this header should read:
"Experiment visualization and metric logging"